packages/yoga/src/Divider/native/Divider.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import theme from '../../Theme/helpers/themeReader';
+
+const StyledDivider = styled.View`
+  ${props => {
+    const { borders, colors } = theme(props);
+
+    return css`
+      height: ${borders.small};
+      width: 100%;
+      background-color: ${colors.light};
+    `;
+  }}
+`;
+
+const DividerContainer = styled.View`
+  flex-direction: row;
```

Comment:
Do we need to set the `display` to `flex`? There are any possible `system` props that could be used here?